core/drivers/pl022_spi.c

```diff
@@ -365,6 +365,12 @@ static void pl022_flush_fifo(struct pl022_data *pd)
 	} while (io_read32(pd->base + SSPSR) & SSPSR_BSY);
 }
 
+static void pl022_flushfifo(struct spi_chip *chip)
+{
+	struct pl022_data *pd = container_of(chip, struct pl022_data, chip);
+	pl022_flush_fifo(pd);
+}
```

Comment:
Since `pl022_flush_fifo()` is only called by `pl022_configure()`, how about just changing `static void pl022_flush_fifo(struct pl022_data *pd)` to `static void pl022_flush_fifo(struct spi_chip *chip)` and registering it with `static const struct spi_ops pl022_ops` so that we don't need to use a wrapper function?

Reply:
Thanks vchong. I will make the change and send new iteration of changes.